variables.tf

```diff
@@ -49,6 +49,26 @@ variable "github_token" {
   description = "(Optional) GitHub auth token environment variable (`GITHUB_TOKEN`)"
 }
 
+variable "jenkins_url" {
```

Comment:
this is related to Jenkins, it should not be in this module which is supposed to be generic